com.civicrm.hrjobroles/gulpfile.js

```diff
@@ -1,21 +1,28 @@
+var exec = require('child_process').exec;
+var civicrmScssRoot = require('civicrm-scssroot')();
+var fs = require('fs');
 var gulp = require('gulp');
 var clean = require('gulp-clean');
 var rename = require('gulp-rename');
 var replace = require('gulp-replace');
 var bulk = require('gulp-sass-bulk-import');
 var sass = require('gulp-sass');
 var karma = require('karma');
-var exec = require('child_process').exec;
 var path = require('path');
-var fs = require('fs');
 
-gulp.task('sass', function (done) {
-  // The app style relies on compass's gems, so we need to rely on it
-  // for the time being
-  exec('compass compile', function (_, stdout, stderr) {
-    console.log(stdout);
-    done();
-  });
+gulp.task('sass', ['sass:sync'], function () {
```

Comment:
Was thinking of having some npm package under the compucorp's github user that we can then declare as a dependency, or something like that. But yeah we'll have to figure out something to avoid this copy-paste